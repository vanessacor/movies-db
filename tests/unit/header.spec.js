import { shallowMount } from "@vue/test-utils";

import Header from "@/components/AppHeader";

describe("test Header.vue component", () => {
  it("matches the snapshot", () => {
    const wrapper = shallowMount(Header, {
        stubs: ['router-link', 'router-view']
    });
    expect(wrapper).toMatchSnapshot();
  });
});
