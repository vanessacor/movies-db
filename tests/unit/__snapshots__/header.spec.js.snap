// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Header.vue component matches the snapshot 1`] = `
<header id="appHeader">
  <h1>Movies DB</h1>
  <nav>
    <router-link-stub to="/">Home</router-link-stub> |
    <router-link-stub to="/movies">Movies</router-link-stub>
  </nav>
</header>
`;
